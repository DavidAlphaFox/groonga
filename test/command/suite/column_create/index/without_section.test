table_create Memos TABLE_NO_KEY
column_create Memos content COLUMN_VECTOR ShortText
load --table Memos
[
{"content": ["Groonga", "Mroonga", "PGroonga"]},
{"content": ["Hello", "Good-bye"]}
]

table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer TokenBigram   --normalizer NormalizerAuto
column_create Terms memos_content COLUMN_INDEX|WITH_POSITION Memos content
