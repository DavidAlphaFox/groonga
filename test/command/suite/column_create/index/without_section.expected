table_create Memos TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Memos content COLUMN_VECTOR ShortText
[[0,0.0,0.0],true]
load --table Memos
[
{"content": ["Groonga", "Mroonga", "PGroonga"]},
{"content": ["Hello", "Good-bye"]}
]
[[0,0.0,0.0],2]
table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer TokenBigram   --normalizer NormalizerAuto
[[0,0.0,0.0],true]
column_create Terms memos_content COLUMN_INDEX|WITH_POSITION Memos content
[
  [
    [
      -22,
      0.0,
      0.0
    ],
    "grn_obj_set_info(): GRN_INFO_SOURCE: full text index for vector column must be created with WITH_SECTION flag: <Terms.memos_con"
  ],
  false
]
#|e| grn_obj_set_info(): GRN_INFO_SOURCE: full text index for vector column must be created with WITH_SECTION flag: <Terms.memos_content>
